Command to import/export taxonomies; AfricanLII base taxonomies

longhotsummer opened the pull request.

actlikewill commented:
@longhotsummer I keep getting the error below when I run webpack. I have updated node packages

![image](https://user-images.githubusercontent.com/25079238/236384614-21d815be-ae57-4f08-a5a6-a889c8c66f23.png)

actlikewill commented:
@longhotsummer Also the child taxonomy links in the taxonomy tree don't work correctly, they appear as: `taxonomy/collections/collections-child/collections-child-child` where `taxonomy/collections/collections-child-child` would suffice.

longhotsummer commented:
> @longhotsummer Also the child taxonomy links in the taxonomy tree don't work correctly, they appear as: `taxonomy/collections/collections-child/collections-child-child` where `taxonomy/collections/collections-child-child` would suffice.

This is now fixed.

longhotsummer commented:
> @longhotsummer I keep getting the error below when I run webpack. I have updated node packages
> 
> ![image](https://user-images.githubusercontent.com/25079238/236384614-21d815be-ae57-4f08-a5a6-a889c8c66f23.png)

This is now fixed (law-widgets 1.4.0 must be installed, so run `npm i`)

longhotsummer commented:
@nickmwangemi and @actlikewill this is ready for review. I want to wait until #1149 is merged before merging, since that will require this to be rebased a bit.

actlikewill reviewed (approved):
Works well. :+1:  searching using taxonomies, awesome!
